### PR TITLE
First try at updating gdl port to 1.0.0.rc1 

### PR DESCRIPTION
**Pull request by slayoo**
Untested - no access to Apple hardware
@tenomoto could you help?

### CODE REVIEW

File: math/gnudatalanguage/Portfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ PortGroup                   mpi 1.0
 PortGroup                   github 1.0
 PortGroup                   cxx11 1.1
 
-github.setup                gnudatalanguage gdl 0.9.9 v
+github.setup                gnudatalanguage gdl 1.0.0-rc.1 v
 revision                    3
```
Comment (mf2k) [inline math/gnudatalanguage/Portfile]: When updating the `version`, the `revision` needs to be reset to 0.

File: math/gnudatalanguage/Portfile [hunk 2]
```diff
@@ -23,18 +23,15 @@ description                 a free IDL combatible incremental compiler
 long_description \
     A free IDL (Interactive Data Language) compatible \
     incremental compiler (i.e. runs IDL programs).
-master_sites                ${github.homepage}/archive:gdl \
-                            http://www.soest.hawaii.edu/pwessel/gshhg:gshhs
+master_sites                ${github.homepage}/archive:gdl
```
Comment (mf2k) [inline math/gnudatalanguage/Portfile]: Normally the GitHub portgroup sets this properly. You may need to add the line:

`github.tarball_from archive
`

**Comment by mf2k** [comment]
Error is during the patch phase:

```
Reversed (or previously applied) patch detected!  Assume -R? [n] 
Apply anyway? [n] 
Skipping patch.
1 out of 1 hunk ignored -- saving rejects to file src/ssrfpack.c.rej

```

File: math/gnudatalanguage/Portfile (hunk 1 above)
Comment (reneeotten) [reply math/gnudatalanguage/Portfile]: I think the policy is to only update to stable, released versions; if you want to go for a pre-release versions that should likely be in a ```-devel``` port. Why not wait until  the final v1.0.0 is released upstream?

Comment (slayoo) [reply math/gnudatalanguage/Portfile]: The 0.9.9 is from Nov 2018, I assume we will have good few subsequent 1.0.0 release candidates, and actually it was our intention to gather feedback from users and packagers before shipping a 1.0.0

**Comment by tenomoto** [comment]
@slayoo is this RC close to production?

**Comment by slayoo** [comment]
we will have a rc-2 within days/weeks and probably a few more rc-s before delivering 1.0.0

**Comment by slayoo** [comment]
@tenomoto @mf2k @cjones051073 (CC @GillesDuvert , @acoulais)
How to proceed? 
I don't really understand the timeout issue
We already have a subsequent release out for some time
How to help to get the portfile updated?

**Comment by cjones051073** [comment]
Please note this PR now has conflicts against master that need to be resolved.

Also note, the git commit messages need to be squashed and also need to follow our guidelines. i.e. the message must take the form `port name: description`.

File: math/gnudatalanguage/Portfile (hunk 2 above)
Comment (GillesDuvert) [reply math/gnudatalanguage/Portfile:26]: wy not just do that? (somebody with knowledge= not me).

**Comment by GillesDuvert** [comment]
> Please note this PR now has conflicts against master that need to be resolved.
> 
> Also note, the git commit messages need to be squashed and also need to follow our guidelines. i.e. the message must take the form `port name: description`.

ok for the message, no probs, but where is the conflict in Portfile?

**Comment by reneeotten** [comment]
Resolving conflicts and rewording commit messages are all trivialities that can be easily done. The time-out issues are just because Travis CI has a maximum of 50 minutes and the port build is  apparently not finished within this limit (it finishes on the Azure pipelines, so this can be ignored). I think the main point here is that there is no new stable release to update to.

**Comment by cjones051073** [comment]
> > Please note this PR now has conflicts against master that need to be resolved.
> > Also note, the git commit messages need to be squashed and also need to follow our guidelines. i.e. the message must take the form `port name: description`.
> 
> ok for the message, no probs, but where is the conflict in Portfile?

the conflict is against the current master, due to a change there. See the summary below. You need to rebase your branch against master, make sure there are no conflicts (if there are fix them) then force push back here.

File: math/gnudatalanguage/Portfile (hunk 2 above)
Comment (mf2k) [reply math/gnudatalanguage/Portfile:26]: Maybe I don't have the time or interest to change it and test it (I don't). You are the submitting the PR and are expected to make the changes and do the testing. That is if you want my commit approval. Of course any other committer can commit it without me.

Comment (GillesDuvert) [reply math/gnudatalanguage/Portfile:26]: @mf2k , I did not mean you...

**Comment by slayoo** [comment]
Let me then close this one and create a new PR for 1.0.0.rc2

**Comment by cjones051073** [comment]
Just for future reference closing this PR was not necessary. You could have made the changes to update to the new RC in the sae branch as used here, and just force pushed them here.

**Pull request closed without merging**